src/selling-partner-api-regions/index.ts

```diff
@@ -7,3 +7,8 @@ export const sellingPartnerRegions = {
   FE,
   NA,
 } as const
+
+export const sellerCentralUrls = {
+  NA: `https://sellercentral.amazon.com`,
```

Comment:
This is not actually entirely correct.

Seller Central is the backend for Amazon Sellers.

Which is not regional, or at least not always regional. It is mostly mapped to a specific country.

E.g.:

 * Canada: https://sellercentral.amazon.ca
 * USA: https://sellercentral.amazon.com
 * UK: https://sellercentral.amazon..co.uk
 * Spain: https://sellercentral.amazon.es

`https://sellercentral-europe.amazon.com` is a regional URL for OAuth only. I don't think it reflects every country.

Also, we used `URI` terminology already, so we should probably stick to that.

Reply:
I can rename it to sellerCentralAuthUri, it would make more sense

Reply:
Ok, let's do that